### PR TITLE
Replace unpack step

### PR DESCRIPTION
**Pull request by LukasKuzavas**
ATTENTION! NEW DOCS STRUCTURE
FOR THE NEXT THREE WEEKS, ONLY BRANCH FROM new-docs-structure
OTHERWISE, WAIT UNTIL THE NEW STRUCTURE HAS BEEN RELEASED

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Adding mention to project templates, deleting projects and syncing projects.

Make changes from unpack steps to open in current projects.

__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

Have you done a trial build to check all new or changed links?

Is there anything else you would like us to know?

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: docs/001_develop/01_development-environment/002_launchpad/index.mdx [hunk 1]
```diff
@@ -283,28 +283,20 @@ This does involve some extra set-up, however.
 ## Genesis Cloud Workspace
 
 When you click on `Open in Genesis Cloud Workspace` in GenesisCreate, your code is generated automatically and placed in a zip file.
```
Comment (rafaelnferreira) [inline docs/001_develop/01_development-environment/002_launchpad/index.mdx:285]: this text has changed I'm sure.

Comment (rafaelnferreira) [reply docs/001_develop/01_development-environment/002_launchpad/index.mdx:285]: (The button)

File: docs/001_develop/01_development-environment/002_launchpad/index.mdx [hunk 2]
```diff
@@ -440,6 +432,10 @@ Then as soon as you start to type Genesis, the available commands are displayed
 | `Stop processes`      | Kills all running Genesis processes on the JVM or server and the WebUi. Use this command when processes are unresponsive. |
 | `Unpack project`      | Unpacks an available project generated in Genesis Create, and makes it active in the workspace. |
 
+:::warning
+`Unpack project` is deprecated and will be removed in the future.
```
Comment (rafaelnferreira) [inline docs/001_develop/01_development-environment/002_launchpad/index.mdx:436]: "is deprecated and no longer necessary, it is currently disabled by default but can be displayed back via settings".

File: docs/001_develop/01_development-environment/002_launchpad/index.mdx [hunk 3]
```diff
@@ -440,6 +432,10 @@ Then as soon as you start to type Genesis, the available commands are displayed
 | `Stop processes`      | Kills all running Genesis processes on the JVM or server and the WebUi. Use this command when processes are unresponsive. |
 | `Unpack project`      | Unpacks an available project generated in Genesis Create, and makes it active in the workspace. |
```
Comment (rafaelnferreira) [inline docs/001_develop/01_development-environment/002_launchpad/index.mdx:434]: Add documentation for Genesis Sync command, I will let you discover what it does.

**Comment by rafaelnferreira** [comment]
Add instructions to Delete a project;
Add instructions to create the project from a template;

**Review by wjhendry — APPROVED** [review]

**Pull request merged**